packages/amplify_datastore/example/ios/unit_tests/AmplifyModelSchemaUnitTests.swift

```diff
@@ -19,46 +19,95 @@ import Amplify
 @testable import amplify_datastore
 
 class AmplifyModelSchemaUnitTests: XCTestCase {
-    
-    let schemasMap: [String: Any] = try! readJsonMap(filePath: "model_schema_maps")
+
+    let modelSchemaMap: [String: Any] = try! readJsonMap(filePath: "model_schema_maps")
+    let customTypeSchemaMap: [String: Any] = try! readJsonMap(filePath: "custom_type_schema_maps")
+    let customTypeSchemasRegistry = FlutterSchemaRegistry()
+
+    override func setUpWithError() throws {
+        // register CustomType Schemas following dependencies order
+        // This also tests the desrialization fucntionlity for CustomType schemas
```

Comment:
Nit: Not sure if this is similar to some of the other changes made
```suggestion
        // This also tests the deserialization functionality for CustomType schemas
```